src/cf-grid/src/cf-grid.less

```diff
@@ -60,14 +59,6 @@
   .lt-ie8 & {
     // Hack inline-block into submission
     display: inline;
-    // Mixing negative margins and the box-sizing hack can be dangerous.
-    // This is a temporary fix for: https://github.com/cfpb/cf-grid/issues/17
-    margin-right: 0;
-
-    // Clearfix hack for IE 6/7 only
-    zoom: 1;
```

Comment:
No, just being careful because the comment didn't mention the polyfill like the other changes. Maybe @Scotchester does?